src/devices_models/device_constructors/hydrogeneration_constructor.jl

```diff
@@ -764,7 +785,8 @@ function construct_device!(
         S,
         get_feedforward(model),
     )
-    commit_hydro_active_power_ub!(container, devices, model, get_feedforward(model))
+    # TODO: check with jose if this being handled by the above add_constraints function is the right thing to do
+    # commit_hydro_active_power_ub!(container, devices, model, get_feedforward(model))
```

Comment:
@sourabhdalvi can you confirm which constraints are needed for the hydro formulations?